### PR TITLE
New dashboard - Investment details

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change
In the "My projects" section of the new dashboard we need to surface the details of each investor.

Note: I have also shifted over some test files into the new "dashboard-new" folder as these were in the wrong place.

## Test instructions
You will need to run the new dashboard, for instructions see #3248

## Screenshots
Screenshot 2021-03-19 at 08 21 03

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: test/functional/cypress/specs/dashboard-new/investment-details.spec.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const { companies, interactions } = require('../../../../../src/lib/urls')
+
+describe('Dashboard - Investment details', () => {
```
Comment (tomkingston) [inline test/functional/cypress/specs/dashboard-new/investment-details.spec.js:3]: It is probably worth adding in another test context where the latest interaction is null. And maybe another for when the country of origin is null too.

**Review by tomkingston — APPROVED** [review]
Approved, but would be good to have another test context for when the last interaction is null

File: src/client/components/MyInvestmentProjects/InvestmentDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <li>
+          <StyledSpan>Country of origin: </StyledSpan>
+          {countryOrigin.name}
+        </li>
+      )}
+      {latestInteraction && (
+        <>
+          <li>
+            <StyledSpan>Last interaction: </StyledSpan>
+            {format(new Date(parseISO(latestInteraction.date)), 'd MMM yyyy')}
```
Comment (paulgain) [inline src/client/components/MyInvestmentProjects/InvestmentDetails.jsx:55]: Any date formatting should go into `/src/common/date.js`

**Review by paulgain — COMMENTED** [review]
LGTM, just a single comment.

File: src/client/components/MyInvestmentProjects/InvestmentDetails.jsx (hunk 2 above)
Comment (ian-leggett) [reply src/client/components/MyInvestmentProjects/InvestmentDetails.jsx:55]: Unfortunately we can't use this as the `date.js` is used on the node layer and this is client side, the file `date.js` uses the commonJS format.

File: test/functional/cypress/specs/dashboard-new/investment-details.spec.js (hunk 1 above)
Comment (ian-leggett) [reply test/functional/cypress/specs/dashboard-new/investment-details.spec.js:3]: Good spot, just updated the tests.

File: src/client/components/MyInvestmentProjects/InvestmentDetails.jsx (hunk 2 above)
Comment (ian-leggett) [reply src/client/components/MyInvestmentProjects/InvestmentDetails.jsx]: As discussed I have used `date-utils.js` instead as this one will format date and time for client side files.

**Review by paulgain — APPROVED** [review]
